include/rdm/responder.h

```diff
@@ -80,6 +90,92 @@ etcpal_error_t rdmresp_unpack_command(const RdmBuffer *buffer, RdmCommand *cmd);
 bool rdmresp_is_non_disc_command(const RdmBuffer *buffer);
 etcpal_error_t rdmresp_pack_response(const RdmResponse *resp_data, RdmBuffer *buffer);
 
+typedef enum
+{
+  kRespAck = E120_RESPONSE_TYPE_ACK,
+  kRespAckTimer = E120_RESPONSE_TYPE_ACK_TIMER,
+  kRespNackReason = E120_RESPONSE_TYPE_NACK_REASON,
+  kRespAckOverflow = E120_RESPONSE_TYPE_ACK_OVERFLOW,
+  kRespNoSend
+} resp_process_result_t;
+
+typedef enum
+{
+  kRespTypeRDM,
```

Comment:
Change `RDM` to `Rdm` to match the rest of our initialism-writing style

Reply:
Done.